meson.build

```diff
@@ -2612,7 +2612,7 @@ if git.found()
                         'ctags',
                         command : [env, 'ctags', '--tag-relative=never', '-o', '@0@/tags'.format(project_source_root)] + all_files)
 
-                ############################################
+        ####################################################
```

Comment:
This one is nested in an `if`, so it should be unindented.

Reply:
@keszybz do you want me to revert this part?